torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -995,10 +995,28 @@ TensorView* sub_alpha(TensorView* v1, TensorView* v2, Val* v3) {
 }
 // lerp
 Val* lerp(Val* start, Val* end, Val* weight) {
+  auto casted_values =
```

Comment:
grammar nit: "cast" is the past tense of "cast"

Reply:
Language is hard. :)